src/ui/app_view.rs

```diff
@@ -129,6 +129,7 @@ pub fn app_view(
 				if key == PhysicalKey::Code(KeyCode::Enter) {
 					{
 						config_search.clone().db.write().add(search_text.get());
+						let _ = config_search.save();
```

Comment:
What do you think about handling these errors, if they ever happen, with a pop up window? Or should we have and error label in the views that need it?